src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -769,8 +769,10 @@ private static string RemoveSxSPsHomeModulePath(string currentProcessModulePath,
         {
 #if UNIX
             const string powershellExeName = "pwsh";
+            const string oldPowershellExeName = "powershell";
 #else
             const string powershellExeName = "pwsh.exe";
+            const string oldPowershellExeName = "powershell.exe";
```

Comment:
Shall we remove the old exe name when we reach RC/RTM?

Reply:
I think this is still needed if we start pwsh from Windows PowerShell

Reply:
I think the "start pwsh from windows powershell" case is handled in [`NeedToClearProcessModulePath`](https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs#L725), and `RemoveSxSPsHomeModulePath` only handles powershell core.

Reply:
Because of the .deps.json file check, this doesn't apply to Windows PowerShell anyways.  We can remove at RC/GA

Reply:
Then I will open an issue on this.